apps/drupal-default/particle_theme/templates/block/block--utc-hero-video--full.html.twig

```diff
@@ -0,0 +1,151 @@
+{#
+/**
+ * @file Video Hero!
+ * Implements a video hero block in dark blue
+ *
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to the cards. 
```

Comment:
Is this comment relevant? I guess this was copied from cards, but I don't think this does mutiple hero or cards?

Reply:
You are correct. It's not relevant. Picked up from previous hero block template and missed on my updates. Will fix both sets.